config.json

```diff
@@ -1116,6 +1116,18 @@
       "unlocked_by": "flatten-array",
       "uuid": "e5f05d00-fe5b-4d78-b2fa-934c1c9afb32"
     },
+    {
+      "core": false,
+      "difficulty": 8,
+      "slug": "diffie-hellman",
+      "topics": [
+        "algorithms",
+        "integers",
+        "transforming"
+      ],
+      "unlocked_by": "book-store",
```

Comment:
The `unlocked_by` should be a core exercise, and `book-store` is not core.

Do you have a suggestion for a suitable value? 🙂

Reply:
I would say that ```linked-list``` is the best candidate 🤔